src/internal/plugins/api.ts

```diff
@@ -58,51 +40,35 @@ function findUpApi(currentWindow: WindowWithApi): AwsuiApi | undefined {
   }
 }
 
-function loadApi() {
+export function loadApi() {
   if (typeof window === 'undefined') {
-    return createApi();
+    return installApi({});
   }
   const win = window as unknown as WindowWithApi;
-  const api = findUpApi(win);
-  if (api) {
-    return api;
-  }
-  win[storageKey] = createApi();
+  const existingApi = findUpApi(win);
+  win[storageKey] = installApi(existingApi ?? {});
```

Comment:
The core part of the fix. We now allow incremental runtime API extension from multiple versioms/bundles